src/main.html

```diff
@@ -110,7 +110,7 @@
     <script type="text/javascript" src="./js/Features.js"></script>
     <script type="text/javascript" src="./js/Beepers.js"></script>
     <script type="text/javascript" src="./js/release_checker.js"></script>
-    <script type="text/javascript" src="./js/jenkins_loader.js"></script>
+    <script type="text/javascript" src="./js/release_loader.js"></script>
```

Comment:
@chmelevskij will be mad that new module does not use modules :)

Reply:
I'm not yet familiar... happy to revert in due course. This was the fastest way home for me.

Reply:
It's ok for now. No need to revert.